extensions/netty/runtime/src/main/java/io/quarkus/netty/runtime/NettyRecorder.java

```diff
@@ -16,17 +16,17 @@
 
     public void eagerlyInitChannelId() {
         //this class is slow to init and can block the IO thread and cause a warning
-        //we init it from a throway thread to stop this
+        //we init it from a throwaway thread to stop this
         //we do it from another thread so as not to affect start time
         new Thread(new Runnable() {
             @Override
             public void run() {
                 long start = System.currentTimeMillis();
                 DefaultChannelId.newInstance();
                 if (System.currentTimeMillis() - start > 1000) {
-                    log.warn(
-                            "Localhost lookup took more than one second, you need to add a /etc/hosts entry to improve Quarkus startup time. "
-                                    + "On Windows the path is C:\\Windows\\System32\\Drivers\\etc\\hosts. See https://thoeni.io/post/macos-sierra-java/ for macOS details.");
+                    log.warn("Netty DefaultChannelId initialization (with io.netty.machineId" +
```

Comment:
I don't know if this is any clearer, as the average user will have no idea what this is about.

With this system property this should no longer be an issue so maybe we should just remove this message?

Reply:
I was thinking of removing the whole `eagerlyInitChannelId()` method from the recorder, now that this system property was set. But given that we haven't seen this change being used in real applications, I wanted to see if even with this system property change, it takes more than a second to initialize it. In which case, we would see users reporting that WARN message.
Maybe we keep this log message for at least 1 release and then if no one complains or reports this message, we remove the whole `eagerlyInitChannelId()` method?

Reply:
Sounds good to me.

Reply:
I just updated this PR to just add a TODO to remove that method in future